core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractLibrary.java

```diff
@@ -539,7 +538,7 @@ public SampleTissue getSampleTissue() {
     if (this.getSample() instanceof DetailedSample) {
 
       for (DetailedSample parent = (DetailedSample) this.getSample(); parent != null; parent = parent.getParent()) {
-        if (parent instanceof SampleTissue) return (SampleTissue) parent;
+        if (parent.getSampleClass().getSampleCategory().equals(SampleTissue.CATEGORY_NAME)) return (SampleTissue) parent;
```

Comment:
This method doesn't seem to be used for anything, so should be removed; otherwise, null checks are needed